connect/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>kafka-stream-skeleton</artifactId>
+        <groupId>com.tikal.org</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>connect</artifactId>
+
+    <repositories>
```

Comment:
Brachi, I think this repository is unnecessary. Try to remove the "repositories" section and see if all works well.

Reply:
no. it's not in maven central repository
```
[ERROR] Failed to execute goal on project connect: Could not resolve dependencies for project com.tikal.org:connect:jar:1.0-SNAPSHOT: Failure to find com.github.confluentinc:kafka-connect-elasticsearch:jar:3.3.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

```

Reply:
OK. Thanks.